src/components/Login/Login.tsx

```diff
@@ -17,17 +17,27 @@ export function Login() {
 		event.preventDefault();
 
 		const authHeader = btoa(`${username}:${password}`);
+
 		try {
 			await fetchLogin(authHeader);
 		} catch (e) {
 			setError((e as Error).message);
 			return;
 		}
-		// const isAdministrator = await isAdmin(username);
+
 		loginContext.logIn(authHeader);
-		// alert to notify user they've successfully logged in
-		console.log("You're logged in");
 		setError(undefined);
+
+		try {
+			await fetchIsAdmin(authHeader);
+		}
+		catch {
+			console.log("You're logged in as a member");
+			return;
+		}
+
+		loginContext.setUserAdmin(true);
+		console.log("You're an admin");
```

Comment:
I think we can call the `fetchIsAdmin` further up and pass this into the `loginContext.logIn` function and set `isAdmin` from there

Reply:
As suggested above